### PR TITLE
fix(UI): Centreon is slowed down

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This fixes Pendo initialization requests flood when the browser is offline.
This also reworks the pendo script to handle it in a React environment 

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

- -> Pendo should send data as before
- Turn off your wifi
- -> Centreon pages are displayed as normal (not slowed down)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/App/usePendo.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import { gte, isNil, not } from 'ramda';
+import { atomWithStorage } from 'jotai/utils';
+import { useAtom } from 'jotai';
+
+import { getData, useRequest } from '@centreon/ui';
+
+import initPendo from './initPendo';
+import { CeipData } from './models';
+
+const oneDay = 24 * 60 * 60 * 1000;
```
Comment (bdauria) [inline www/front_src/src/App/usePendo.ts:12]:
```suggestion
const oneDayInMs = 24 * 60 * 60 * 1000;
```

**Review by bdauria — APPROVED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Review by lpinsivy — CHANGES_REQUESTED** [review]
Pendo needs to be loaded in footerPart.php too due to iframe!

File: www/front_src/src/App/usePendo.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    } catch (e) {
+      setIsCeipEnabled(true);
+    }
+  };
+
+  const online = (): void => {
+    sendCeipInfo(true);
+  };
+
+  const offline = (): void => {
```
Comment (bdauria) [inline www/front_src/src/App/usePendo.ts:89]: function names should be verb (goOnline / goOffline would do for instance)

Comment (bdauria) [inline www/front_src/src/App/usePendo.ts]: function names should be verb (goOnline / goOffline would do for instance)

**Review by bdauria — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Pull request merged**